src/doc/data_into_visit/SiloFormat.rst

```diff
@@ -1,6 +1,1621 @@
+
+.. code-block::
+
+  PLATFORM=i386-apple-darwin10_gcc-4.2
+  SZIP_DIR=/usr/local/szip/2.1/$(PLATFORM)
+  SZIP_CPPFLAGS=-I$(SZIP_DIR)/include
+  SZIP_LDFLAGS=-L$(SZIP_DIR)/lib
+  SZIP_LIBS=-lsz
+  HDF5_DIR=/usr/local/hdf5/1.8.4/$(PLATFORM)
+  HDF5_CPPFLAGS=-I$(HDF5_DIR)/include $(SZIP_CPPFLAGS)
+  HDF5_LDFLAGS=-L$(HDF5_DIR)/lib $(SZIP_LDFLAGS)
+  HDF5_LIBS=-lhdf5 $(SZIP_LIBS) -lz
+  SILO_DIR=/usr/local/silo/4.6.2/$(PLATFORM)
+  SILO_CPPFLAGS=-I$(SILO_DIR)/include $(HDF5_CPPFLAGS)
+  SILO_LDFLAGS=-L$(SILO_DIR)/lib $(HDF5_LDFLAGS)
+  SILO_LIBS=-lsiloh5 $(HDF5_LIBS) -lm
+  LDFLAGS=$(LDFLAGS) $(SILO_LDFLAGS)
+  LIBS=$(SILO_LIBS)
+  CPPFLAGS=$(CPPFLAGS) $(SILO_CPPFLAGS)
+
+If your *Makefile* does not use *CPPFLAGS* then you might try adding the `-I` include directives to *CFLAGS*, *F77FLAGS*, or whichever make variables are relevant for your Makefile.
+
+Using Silo on Windows
+"""""""""""""""""""""
+
+When you build an application using the Silo_ library on Windows, you can use the precompiled Silo_ DLL and import library that comes with the VisIt_ development distribution for Windows: *visit_windowsdev_x.y.x.zip*, where *x.y.z* refers to the version, like *3.3.3*.
+The development distribution for Windows includes pre-built binaries (.dlls and import libraries) for the Third party libraries upon which VisIt_ depends, including Silo_.
+Simply unzip the distribution to whichever location best suits your needs.
+The binaries are located in the *windowsbuild/MSVC<VERSION>* folder, with *<VERSION>* being the version of Visual Studio they were built with (eg MSVC2017).
+
+If you want to build an application against the Silo_ library provided with VisIt_, add the path to silo.h to your project file.
+After setting the Silo_ include directory to your project file, make sure that the Silo_'s import library is in your linker path.
+Next, add silohdf5.lib (or siloh5.lib, depending on the version of Silo_) to the list of libraries that are linked with your program.
+That should be enough to get your program to build.
+
+Before running your program, be sure to copy *silohdf5.dll*, *hdf5dll.dll*, *sziplib.dll*, and *zlib.dll* from the VisIt windows distribution into the directory where your program will execute.
+Note that you must configure your program to use a Multithreaded DLL version of the Microsoft runtime library or using the precompiled Silo_ library may result in fatal errors.
+
+Inspecting Silo files
+~~~~~~~~~~~~~~~~~~~~~
+
+Silo_ includes a command line utility called *browser* that can access the contents of Silo_ files.
+To run browser, type *"browser"* into a terminal window followed by the name of a Silo_ file that you want to inspect.
+Once the browser application opens the Silo_ file, type *"ls"* to see the contents of the Silo_ file.
+From there, typing the name of any of the objects shown in the object listing will print information about that object to the console.
+
```

Comment:
Should we add a mention of Silex too?

Reply:
Feel free, (or we can create a ticket to add it later). It's been a long time since I have used silex myself and don't know if I could adequately describe it.

Reply:
Ok, I propose just a minor additional line here.